src/WeekFour/HomeworkRecursion.java

```diff
@@ -0,0 +1,69 @@
+package WeekFour;
+
+import java.io.PrintStream;
+
+public class HomeworkRecursion {
+    /*
+    *   Algorithms Homework – Recursion
+    *   Recursion
+    *   Solve FizzBuzz and Palindrome exercise using recursion.
+    * */
+    public static void main(String[] args){
+
+        //Recursion Palindrome
+        System.out.println("Recursion Palindrome:");
+        System.out.println("Racecar" + " Is Palindrome ? " + isPalindrome("Racecar"));
+        System.out.println("dad" + " Is Palindrome ? " + isPalindrome("dad"));
+        System.out.println("Was It A Cat I Saw?" + " Is Palindrome ? " + isPalindrome("Was It A Cat I Saw?"));
+
+        //Recursion Fizzbuzz
+        System.out.println("\nRecursion Fizzbuzz:");
+        fizzBuzz(1,15);
+    }
+
+    public static boolean isPalindrome(String word){
```

Comment:
I think you could separate this method on two methods. First method would cast a string to required state and then second method would verify a string on a palindorm using a recursion